### PR TITLE
Generate user permissions from gsheet ansibled roger 888

### PR DESCRIPTION
**Pull request by ankanm**
- [x] Move script inside ansible files.
- [x] Add ability to specify output filename.
- [x] Generate user permissions locally, send to remote and delete the local file.
- [x] ~~Figure out how best to deal (set up) the oauth file~~ Templatize oauth and move secrets to variables.
- [x] Remove the remote rsync that fails sometimes.

@vmahedia

### CODE REVIEW

File: ansible/roles/marathon-server/tasks/main.yml [hunk 1]
```diff
@@ -84,6 +84,45 @@
       - plugin
       - configuration
 
+  - name: Create a temp local permissions file when marathon_auth_permissions_gspreadsheets is true
+    when: marathon_auth_enable is defined and marathon_auth_enable|bool and
+          marathon_auth_permissions_gspreadsheets is defined and not(marathon_auth_permissions_gspreadsheets | trim == '')
```
Comment (vmahedia) [inline ansible/roles/marathon-server/tasks/main.yml]: What does this `trim == ' '` thing do?

**Comment by vmahedia** [comment]
LGTM

File: ansible/roles/marathon-server/tasks/main.yml (hunk 1 above)
Comment (ankanm) [inline ansible/roles/marathon-server/tasks/main.yml]: `var | trim` does a strips leading & trailing spaces in `var`

**Comment by vmahedia** [comment]
LGTM

**Pull request merged**